pkgs/by-name/pa/parabolic/package.nix

```
@@ -40,6 +40,10 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-B8/e5urhy5tAgHNd/PR3HlNQd0M0CxgC56nArFGlQ9c=";
   };
 
+  patches = [
+    ./wrap-workaround.patch
+  ];
```

I think this would be a better way to go about it, as the larger issue here is https://github.com/NixOS/nixpkgs/issues/308324

This also avoids a full patch file and changing the name of the binary (which is something that may be kinda confusing IMO)

```suggestion
  postPatch = ''
    for file in resources/linux/org.nickvision.tubeconverter.{desktop,service}.in; do
      substituteInPlace "$file" \
        --replace-fail '@CMAKE_INSTALL_FULL_LIBDIR@/@PROJECT_NAME@/' ''
    done
  '';
```